src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ConstantNameCheck.java

```diff
@@ -73,16 +104,15 @@
  * <p>Code Example:</p>
  * <pre>
  * class MyClass {
- *   final static int log = 10; // OK
- *   final static int logger = 50; // OK
- *   final static int logMYSELF = 10; // violation, name 'logMYSELF' must match
- *                                    // pattern '^log(ger)?$|^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
- *   final static int loggerMYSELF = 5; // violation, name 'loggerMYSELF' must match
- *                                      // pattern '^log(ger)?$|^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
- *   final static int MYSELF = 100; // OK
- *   final static int myselfConstant = 1; // violation, name 'myselfConstant' must match pattern
- *                                        // '^log(ger)?$|^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
- * }
+ * final static int log = 10; // OK
```

Comment:
These lines don't look like they should have been changed. All I see is indentation being removed and no reason why it should have.